packages/react-components/react-dialog/etc/react-dialog.api.md

```diff
@@ -117,7 +120,7 @@ export type DialogOpenChangeEventHandler = (event: DialogOpenChangeEvent, data:
 // @public (undocumented)
 export type DialogProps = ComponentProps<Partial<DialogSlots>> & {
     modalType?: DialogModalType;
-    open?: boolean;
+    open?: MotionShorthand<DialogSurfaceElement>;
```

Comment:
While we can use the motion hook internally in the Dialog, we should not make its public API the Dialog's public API - that is still a preview package, and we would be essentially killing any possibility of major bumps there in the future